packages/grid/_modules_/grid/components/cell/GridEditBooleanCell.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import Checkbox, { CheckboxProps } from '@material-ui/core/Checkbox';
+import { GRID_CELL_EDIT_PROPS_CHANGE } from '../../constants/eventsConstants';
+import { GridCellParams } from '../../models/params/gridCellParams';
+
+export function GridEditBooleanCell(props: GridCellParams & CheckboxProps) {
+  const {
+    id: idProp,
+    value,
+    element,
+    formattedValue,
+    api,
+    field,
+    row,
+    colDef,
+    cellMode,
+    getValue,
+    rowIndex,
+    colIndex,
+    isEditable,
+    ...checkboxProps
```

Comment:
I assume it's related to #1397

```suggestion
    ...other
```